### PR TITLE
Configure and deploy app to Firebase

### PR DESCRIPTION
**Pull request by merecka**
Configured and deployed the app to Firebase.

### CODE REVIEW

File: .firebase/hosting.ZGlzdA.cache [hunk 1]
```diff
@@ -0,0 +1,11 @@
+oshop-dashboard/3rdpartylicenses.txt,1583782226617,f16cf111afc8a335a9e325d14500a73a1ee303381c7548a926fc12ac28569ae4
```
Comment (ahaly92) [inline .firebase/hosting.ZGlzdA.cache:1]: this file should not be checked in

File: .firebase/hosting.ZGlzdC9vc2hvcC1kYXNoYm9hcmQ.cache [hunk 2]
```diff
@@ -0,0 +1,11 @@
+3rdpartylicenses.txt,1583782226617,f16cf111afc8a335a9e325d14500a73a1ee303381c7548a926fc12ac28569ae4
```
Comment (ahaly92) [inline .firebase/hosting.ZGlzdC9vc2hvcC1kYXNoYm9hcmQ.cache:1]: same here

File: dist/oshop-dashboard/index.html [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```
Comment (ahaly92) [inline dist/oshop-dashboard/index.html:1]: dist directories get generate when you build a project. They should be gitignored. add the directory to your git ignore

File: src/app/admin/admin-orders/admin-orders.component.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+@Component({
+  selector: 'app-admin-orders',
+  templateUrl: './admin-orders.component.html',
+  styleUrls: ['./admin-orders.component.scss']
+})
+export class AdminOrdersComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit(): void {
```
Comment (ahaly92) [inline src/app/admin/admin-orders/admin-orders.component.ts]: this is not used so we should remove it and not have the component implement OnInit

Comment (merecka) [reply src/app/admin/admin-orders/admin-orders.component.ts]: Do you mean that I will not use the entire component at all for the whole project?

Comment (ahaly92) [reply src/app/admin/admin-orders/admin-orders.component.ts]: just the ngOnInit function. When you implement OnInit for a component you need to  add the ngOnInit lifecycle function but here you have no logic in that function so you can rip it out for simplicity.

Comment (merecka) [reply src/app/admin/admin-orders/admin-orders.component.ts]: OK got it.  Thanks for the explanation.

**Pull request closed without merging**